src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -675,7 +678,7 @@ internal bool TryReadInt64(out long value)
                 // then use ReadByteArray since the logic is there to take care of that.
 
                 int bytesRead;
-                if (!TryReadByteArray(_bTmp.AsSpan(start: _bTmpRead), 8 - _bTmpRead, out bytesRead))
+                if (!TryReadByteArray(_bTmp.AsSpan(_bTmpRead), 8 - _bTmpRead, out bytesRead))
```

Comment:
I added the parameter name because it can be confusing to look at these AsSpan calls because it isn't clear what the parameter means, is it the length of bytes or the start? I thought just adding this one made reasonably clear that we're taking a slice by `_bTmp[_bTmpRead..(8 - _btmpRead)]`

They're not needed and have no cost so it'd be nice to keep them for readability.

Reply:
Reversed the merge and added the param name to the netcore code.